### PR TITLE
Aaron/fix ut

### PR DESCRIPTION
**Pull request by AaronZgl**
- Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/okex/okexchain/blob/develop/CONTRIBUTING.md#pr-targeting))

- [ ] Linked to github-issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote tests
- [ ] Updated relevant documentation (`docs/`)
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Reviewed `Files changed` in the github PR explorer

______

For Admin Use:
- Added appropriate labels to PR (ex. wip, ready-for-review, docs)
- Reviewers Assigned
- Squashed all commits, uses message "Merge PR #XYZ: [title]" ([coding standards](https://github.com/okex/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: x/token/token_test.go [hunk 1]
```diff
@@ -747,13 +748,13 @@ func TestCreateMsgMultiSend(t *testing.T) {
...
 		app.supplyKeeper,
 		auth.DefaultSigVerificationGasConsumer,
-		func(ctx sdk.Context, msgs []sdk.Msg) sdk.Result {
-			return sdk.Result{}
-
-		},
-		func(ctx sdk.Context, msgs []sdk.Msg) bool {
-			return false
-		},
+		//func(ctx sdk.Context, msgs []sdk.Msg) sdk.Result {
```
Comment (zhongqiuwood) [inline x/token/token_test.go:751]: remove them

File: x/token/token_test.go [hunk 2]
```diff
@@ -891,13 +892,13 @@ func getMockAppToHandleFee(t *testing.T, initBalance int64, numAcc int) (app *Mo
...
```
Comment (zhongqiuwood) [inline x/token/token_test.go:895]: remove them

**Review by zhongqiuwood — APPROVED** [review]

File: x/token/token_test.go (hunk 1 above)
Comment (AaronZgl) [reply x/token/token_test.go]: done

File: x/token/token_test.go (hunk 2 above)
Comment (AaronZgl) [reply x/token/token_test.go]: done

**Review by ilovers — APPROVED** [review]

**Pull request merged**